File: Activity54.c
#include <stdio.h>

int main() {
    int base, exponent;
    long long result = 1;

    // Ask for base
    printf("Enter the base number: ");
    scanf("%d", &base);

    // Ask for exponent
    printf("Enter the exponent: ");
    scanf("%d", &exponent);

    // Only handle non-negative exponents
    if (exponent < 0) {
        printf("Negative exponents not supported.\n");
        return 0;
    }

    // Multiply base exponent times
    while (exponent > 0) {
        result *= base;
        exponent--;
    }

    // Print result
    printf("Output: %lld\n", result);

    return 0;
}

File: Java/Activity54.java
import java.util.Scanner;

public class Activity54 {

    public static int power(int base, int exponent) {
        int result = 1;
        for(int i = 1; i <= exponent; i++) {
            result *= base;
        }
        return result; 
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Enter base: ");
        int base = in.nextInt();

        System.out.print("Enter exponent: ");
        int exponent = in.nextInt();

        int output = power(base, exponent);

        System.out.println("Output: " + output);
    }
}
